tools/benchmark_tool/pyproject.toml

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Use this configuration file to create a wheel with OpenVINO™ Python* benchmark tool:
+# $ python -m pip wheel . --no-deps
+
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "benchmark_tool"
+version = "0.0.0"
+description = "OpenVINO™ Python* tools package"
```

Comment:
```suggestion
description = "OpenVINO™ Python* tools package"
```
shouldn't it be changed as openvino_dev will be dropped?
CC @ilya-lavrenov

Reply:
changed to what?

Reply:
Will it be a part of the openvino wheel or separate benchmark_tool wheel?

Reply:
part of openvino as of now

Reply:
then we should use openvino's pyproject.toml instead, no?

Reply:
@mryzhov generally yes, but we wanted to split the whole migration to the small steps...